### PR TITLE
Remove all external references

### PR DESCRIPTION
**Pull request by elliottomlinson**
as per #47 

**Action Names**
- [x] Bound Bow
- [x] Bound Sword
- [x] Clippy
- [x] Doorstucker
- [x] Ender Pearl
- [x] Flaming Familiar
- [x] Frostbite Familiar
- [x] Mage Hand
- [x] Monk Mode
- [x] Proplauncher 
- [x] The Queen's Gambit
- [x] Shockwave Familiar
- [x] Slag Familiar
- [x] Woody's Roundup

**Card Illustrations**
- [x] ~Bound Bow~ Bow Charm
- [x] ~Bound Sword~ Sword Charm
- [x] ~Clippy~ Word Warp
- [x] Cowboy Reload
- [x] ~Doorstucker~ Porticostatic Hex
- [x] Empouched Earth
- [x] ~Ender Pearl~ Platzfalt Step
- [x] Fireball
- [x] Grapple
- [x] Gust of Wind
- [x] Nullify Magic
- [x] ~Proplauncher~ Object Eject
- [x] Stormcall

**Rules Text**
- [x] Anvil Drop
- [x] Empouched Earth
- [x] The Swarm
- [x] The Ultimatum
- [x] Doorstucker
- [x] Proplauncher
- [x] The Ultimatum
- [x] Smoke Bomb
- [x] You're Toast
- [x] Whip
- [x] Flirt
- [x] Arouse
- [x] Seduce
- [x] Tactical Reload
- [x] Expert Reload
- [x] Adlib
- [x] Helter-Skelter
- [x] Fork Lightning
- [x] Elektra's Siphon
- [x] Ignite
- [x] Disguise
- [x] Mirror Image

### CODE REVIEW

**Comment by elliottomlinson** [comment]
- [x] Action names

**Changes**:

| Old Name | New Name |
|-|-|
| Bound Bow | Bow Charm |
| Bound Sword | Sword Charm |
| Clippy | Word Warp |
| Doorstucker | Porticostatic Hex |
| Ender Pearl | Platzfalt Step |
| Mage Hand | Eidolimb |
| Monk Mode | Bashbolster |
| The Queen's Gambit | Queen's Gambit |
| Woody's Roundup | Sherrif's Roundup |

**Removals**:

The familiar series has been removed pending a re-work. This resolves:

- Flaming Familiar
- Frostbite Familiar
- Shockwave Familiar
- Slag Familiar

File: data/card_catalogue.json [hunk 1]
```diff
@@ -403,13 +403,13 @@
           }
         },
         {
-          "title": "Monk Mode",
+          "title": "Bashbolster",
```
Comment (DuncanUszkay1) [inline data/card_catalogue.json:406]: What was Monk Mode referencing? DnD?

File: data/card_catalogue.json [hunk 2]
```diff
@@ -403,13 +403,13 @@
...
           "upgrade": "",
           "tier": "green",
           "details": {
             "rules": "Unarmed attacks deal double damage until you play your next card.",
```
Comment (DuncanUszkay1) [inline data/card_catalogue.json:410]: We should change `unarmed` into `melee`. Players can't really be armed

File: data/card_catalogue.json (hunk 1 above)
Comment (elliottomlinson) [reply data/card_catalogue.json:406]: That and some fringe internet groups we may not want to end up associated with

File: data/card_catalogue.json (hunk 2 above)
Comment (elliottomlinson) [reply data/card_catalogue.json:410]: Sounds good, after I'm done with this I'll open an issue about reviewing the language in the catalogue - along with this there are some other semantic inconsitencies (i.e. "square" and "tile") I'd like to tidy up just for the sake of polish.

**Review by DuncanUszkay1 — APPROVED** [review]
LGTM

File: data/card_catalogue.json [hunk 3]
```diff
@@ -1373,7 +1373,7 @@
           "tier": "gold",
           "details": {
             "rules": "Summon a copy of any non-player character",
-            "flavour": "'It's very nice to meet you Aunt Jemima' -Slick Sammy",
```
Comment (elliottomlinson) [inline data/card_catalogue.json:1376]: Sorry Aunt Jemima

**Comment by elliottomlinson** [comment]
- [x] Rules and flavor text

Removed all real names from flavour quote attributions, direct references to other games, etc.

**Comment by elliottomlinson** [comment]
- [x] card illustrations

ready to merge

**Comment by elliottomlinson** [comment]
- [x] card illustrations

ready to merge

**Pull request merged**